build.ps1

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env pwsh
+#Requires -Version 5.1
```

Comment:
Update readme about minimum powershell version and how to install it

Reply:
> Update readme about minimum powershell version and how to install it

PowerShell 5.1 is built into Windows.

Reply:
The readme file says 7.2 so it's not consistent. Also at least which version of Windows comes with the right powershell? e.g., windows 7 has powershell 2.0